src/connectors/infinite-hits/connectInfiniteHits.ts

```diff
@@ -245,6 +247,7 @@ const connectInfiniteHits: InfiniteHitsConnector = function connectInfiniteHits(
 
       getWidgetRenderState({ results, helper, state, instantSearchInstance }) {
         let isFirstPage;
+        let currentPageHits;
```

Comment:
We'd better make this `[]` by default, or to make the type optional, because `getWidgetRenderState` here returns `undefind` as `currentPageHits` when there's no results.